### PR TITLE
Issue #30 - Convert Web API Server tests to Core, and remove Date Par…

### PR DESCRIPTION
**Pull request by darnjo**
See Issue #30 

- Removed references to Gold and Platinum and replaced them with Core
- Removed old RESOScript templates and provided an updated `web-api-server.core.1.0.2.resoscript` template
- Renamed tests and code references to Gold and Platinum accordingly
- Updated Gradle builds
- Updated Tests and test templates
- Removed REQ-WA103-QO18
- Removed REQ-WA103-QO19
- Removed REQ-WA103-QO20
- Removed REQ-WA103-QO21
- Removed REQ-WA103-QO22
- Removed REQ-WA103-QO23
- Removed REQ-WA103-QO24
- Updated README

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -124,60 +96,6 @@ task testWebApiServer_1_0_2_Platinum() {
...
-          'org.reso.certification.stepdefs#DataDictionary',
-          'src/main/java/org/reso/certification/features/data-dictionary',
-          '--tags',
-          (systemProperties.get('cucumber.filter.tags', '').toString().trim().length() > 0
-              ? '@DD1.6 and ' + systemProperties.get('cucumber.filter.tags') : '@DD1.6')
-      ]
-    }
-  }
-}
-
```
Comment (EnFinlay) [inline build.gradle:180]: We don't want to keep these Data Dictionary tasks?

File: src/test/resources/mock.web-api-server.core.1.0.2.resoscript [hunk 2]
```diff
@@ -511,84 +498,6 @@
...
-      OutputFile="REQ-WA103-QO16_ToLowerEQ.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=tolower(*Parameter_StringField*) eq '*Parameter_ToLowerValue*'*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO17"
-      OutputFile="REQ-WA103-QO17_ToUpperEQ.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=toupper(*Parameter_StringField*) eq '*Parameter_ToUpperValue*'*Parameter_RequiredParameters*"
-    />
-
```
Comment (EnFinlay) [inline src/test/resources/mock.web-api-server.core.1.0.2.resoscript:591]: Even though these aren't part of core, is there a good place to keep them for when they get used by endorsements? Or are they already somewhere else in the codebase

File: build.gradle (hunk 1 above)
Comment (darnjo) [reply build.gradle:180]: They will be updated with a DD 1.7 task when the dd-test-generator branch is merged. The ones that were here were for an old set of tests that have changed a bit. Good find though.

File: src/test/resources/mock.web-api-server.core.1.0.2.resoscript (hunk 2 above)
Comment (darnjo) [reply src/test/resources/mock.web-api-server.core.1.0.2.resoscript:591]: I figured we'd just go back in GitHub and grab them when we needed them. They will have their own `.resoscript` files and acceptance test `.feature` files. I'll create an issue so as not to forget.

Comment (darnjo) [reply src/test/resources/mock.web-api-server.core.1.0.2.resoscript:591]: Issue #33 created.

**Pull request merged**